Tasks/Gorbachev/Task_7/data.py

```diff
@@ -0,0 +1,81 @@
+# catalog_dict = {id : [name, [category], product_code, price, stock, rating, sale]}
+catalog_dict = {1 : ["T-short", ["Wear", "Sports"], 991, 99.95, 5, 5.8, False],
+                2 : ["Nike shoes", ["Shoes", "Sports"], 992, 89.70, 2, 8.8, False],
+                3 : ["Keyboard Asus Strix", ["Computer", "Electronics"], 91, 60.00, 5, 9.8, False],
+                4 : ["Videocard Asus 3090 TI", ["Computer", "Electronics"], 15, 25.50, 4, 7.5, True],
+                5 : ["Smart watch Apple Series 5", ["Accessoires", "Electronics"], 425, 152.15, 9, 9.8, False],
+                6 : ["Iphone 13 Pro", ["Smartphones", "Electronics"], 48, 252.15, 15, 6.0, True],
+                7 : ["Sauron ring", ["Jewelry"], 1, 99999.99, 1, 10.0, True],
+                8 : ["Dark Souls 3", ["Games", "Electronics"], 51, 99.99, 10, 2.5, False]}
+
+category_dict = {1 :"Wear",
+                 2: "Sports",
+                 3: "Shoes",
+                 4: "Computer", 
+                 5: "Electronics",
+                 6: "Accessoires",
+                 7: "Smartphones",
+                 8: "Jewelry",
+                 9: "Games"}
+
+basket_dict = {}
+
+def view(basket=None):
+    """Return values from catalog or basket"""
+    if not basket:
+        return catalog_dict.values()
+    if basket:
+        return basket_dict.values()
+    
+def categories_view():
+    """Return categories from catalog"""
+    return category_dict.items()
+
+def category_items(choosen_category):
+    """Return items that belongs to choosen category"""
+    if choosen_category in category_dict:
+        return category_dict[choosen_category]
+    else:
+        return False
+
+def buy_items():
+    """Return empty basket and decreased quantity of items in catalog"""
+    global catalog_dict, basket_dict
+    if basket_dict == {}:
+        return False
+    for y,z in catalog_dict.copy().items():
```

Comment:
are you sure the simple copy is enough?

Reply:
we search in copy of initial catalog. and when quantity of goods is 0 we need to delete all position from initial catalog. if we'll serch in initial catalog in loop we cannot delete position in initial catalog.
maybe there is better solution.